index.html

```diff
@@ -7,37 +7,192 @@
+        <div class="container-image-3">
+        </div>
+          <section class="left-block">                
+              <div class="tonic-title-2">
+                      <h2>Tonic</h2>
+                  <ul class="frame-2">
+                      <li class="canopy-text"><h3>CANOPY</h3></li>
+                      <li class="role-text"><h3>Back End Dev</h3></li>
+                      <li class="year-text"><h3>2015</h3></li>
+                  </ul>
+              </div>
+              <div class="text-paragraph">
+                  <p>
+                      A daily selection of privately personalized reads; 
+                      no accounts or sign-ups required.
+                  </p>
+              </div> 
+              
+              <ul class="primary-buttons">
+                  <li><button class="button1" type="button">html</button></li>
+                  <li><button class="button2" type="button">css</button></li>
+                  <li><button class="button3" type="button">Javascript</button></li>
+              </ul>
+                      <button class="see-project" type="button">See Project</button>
+          </section>
+        </section>
+      
+
+      <section class="tonic-card">
+        <div class="container-image-4">
+        </div>
+          <section class="left-block">                
+              <div class="multi-post">
+                      <h2>Tonic</h2>
+                  <ul class="frame-2">
+                      <li class="canopy-text"><h3>CANOPY</h3></li>
+                      <li class="role-text"><h3>Back End Dev</h3></li>
+                      <li class="year-text"><h3>2015</h3></li>
+                  </ul>
+              </div>
+              <div class="text-paragraph">
+                  <p>
+                      A daily selection of privately personalized reads; 
+                      no accounts or sign-ups required.
+                  </p>
+              </div> 
+              
+              <ul class="primary-buttons">
+                  <li><button class="button1" type="button">html</button></li>
+                  <li><button class="button2" type="button">css</button></li>
+                  <li><button class="button3" type="button">Javascript</button></li>
+              </ul>
+                      <button class="see-project" type="button">See Project</button>
+          </section>
+        </section>
+        
+        <section class="about-me-section">
+            <section>
+                <div class="about-myself">
+                    <h1>About Myself</h1>
+                </div>                
+                <p class="about-me-text">
+                    I’m a software developer! I can help you build a product , 
+                    feature or website Look through some of my work and experience! 
+                    If you like what you see and have a project you need coded, 
+                    don’t hestiate to contact me.
+                </p>

```

Comment:
- [ ] Please work on this paragraph to make it looks like the Figma temple :+1: check the screenshot below and see the differences :smile: 


### Figma template
![Screenshot from 2022-07-07 23-54-41](https://user-images.githubusercontent.com/6615394/177890782-67dbce16-45ef-42c6-b0cc-fc6139831be9.png)

### Your design
![Screenshot from 2022-07-07 23-54-26](https://user-images.githubusercontent.com/6615394/177890788-22a17417-b166-4ac9-aa2b-c864b12ceba8.png)